### PR TITLE
Apache dashboard

### PR DESCRIPTION
**Pull request by JakubKotkowiak**

### CODE REVIEW

File: library/apache/config.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Name of your quickstart as shown on the website
+name: Apache
+description: |+
+  Describe what's the focus of your dashboards.
+  You can use multiple lines.
+  Try to be as descriptive as possible.
+
+# Optional: Authors of the quickstart
+authors:
+    - Newrelic
```
Comment (kidk) [inline library/apache/config.yaml:10]: `New Relic`

File: library/apache/dashboards/apache.json [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+                                "accountId": 0,
+                                "query": "SELECT average(`net.bytesPerSecond`) as 'Bytes Sent' FROM ApacheSample facet entityName TIMESERIES "
+                            }
+                        ]
+                    }
+                }
+            ]
+        }
+    ],
+    "name": "Apache",
```
Comment (kidk) [inline library/apache/dashboards/apache.json:204]: Could you manually move up the name and description field?

**Review by kidk — CHANGES_REQUESTED** [review]
JSON looks good. I've added to small comments. 

Also when you take the screenshot of the dashboards, could you change the time range so it shows data on the whole graph? 

But besides that, everything looks great. Thanks 👍

**Comment by JakubKotkowiak** [comment]
> JSON looks good. I've added two small comments.
> 
> Also when you take the screenshot of the dashboards, could you change the time range so it shows data on the whole graph?
> 
> But besides that, everything looks great. Thanks 👍

The shortest it appears that I can go is 30 minutes, but yeah it's no problem.

**Review by kidk — APPROVED** [review]

**Comment by kidk** [comment]
Looks good 👍

**Pull request merged**